eox_core/api/v1/views.py

```diff
@@ -191,59 +195,202 @@ class EdxappEnrollment(UserQueryMixin, APIView):
+                {
+                  "namespace": "credit",
+                  "name": "provider_id",
+                  "value": "institution_name"
+                }
+              ]
+            }
+
+        **Parameters**
+
+        - `username` (**required**, string, _body_):
+            The username used to identify a user enrolled on the course. Use either username or email.
+
+        - `email` (**required**, string, _body_):
+            The email used to identify a user enrolled on the course. Use either username or email.
+
+        - `course_id` (**required**, string, _body_):
+            The course id for the enrollment you want to update.
+
+        - `mode` (**required**, string, _body_):
+            The course mode for the enrollment. Must be available for the course.
+
+        - `is_active` (boolean, _body_):
+            Flag indicating whether the enrollment is active.
+
+        - `enrollment_attributes` (list, _body_):
+            An enrollment attribute can be used to add extra parameters for a specific course mode.
+            It must be a dictionary containing the following:
+            - namespace: namespace of the attribute
+            - name: name of the attribute
+            - value: value of the attribute
+
+        **Returns**
+
+        - 200: Success, enrollment updated.
+        - 202: User or enrollment doesn't belong to site.
+        - 400: Bad request, invalid course_id or missing either email or username.
         """
         data = request.data
         return EdxappEnrollment.prepare_multiresponse(data, self.single_enrollment_update)
 
+    @apidocs.schema(
+        parameters=[
+            apidocs.query_parameter(
+                name='username',
+                param_type=str,
+                description='**required**, The username used to identify a user enrolled on the course. Use either username or email.'
+            ),
+            apidocs.query_parameter(
+                name='email',
+                param_type=str,
+                description='**required**, The email used to identify a user enrolled on the course. Use either username or email.'
+            ),
+            apidocs.query_parameter(
+                name='course_id',
+                param_type=str,
+                description='**required**, The course id for the enrollment you want to check.'
+            ),
+        ],
+        responses={
+            200: EdxappCourseEnrollmentSerializer,
+            400: 'Bad request, missing course_id or either email or username',
+            404: 'User or course not found',
+        }
+
+    )
     def get(self, request, *args, **kwargs):
         """
-        Get enrollments on edxapp
+        Retrieves enrollment information given a user and a course_id
+
+        **Example Requests**
+
+            GET /eox-core/api/v1/enrollment/?username=johndoe&
+            course_id=course-v1:edX+DemoX+Demo_Course
+
+            Request data: {
```

Comment:
in this request data, according to the documentation, you could pass either the email or username. Maybe that could be specified here somehow?

Reply:
Where would that be? That request data is more like a visual aid because reading the whole string can be a little bit difficult. In that example I'm performing a request with `username`, I didn't want to add more examples because it gets a bit too long and I added a small explanation on the description of the parameters.

Reply:
Yes, you are right. No need to add it here since this is just an example